src/Microsoft.Extensions.Configuration.AzureAppConfiguration/AzureAppConfigurationProvider.cs

```diff
@@ -439,11 +454,11 @@ public async Task<bool> TryRefreshAsync(CancellationToken cancellationToken)
             {
                 if (IsAuthenticationError(e))
                 {
-                    _logger?.LogWarning(e, LoggingConstants.RefreshFailedDueToAuthenticationError);
+                    AzureAppConfigurationProviderEventSource.Log.LogWarning(e, LoggingConstants.RefreshFailedDueToAuthenticationError);

```

Comment:
I'm confused. Are we not logging with ILogger anymore even if it's available?

Reply:
I'll add a helper to structure messages and solve this in those next changes.